spec/classes/dhcp6_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'spec_helper'
+
+describe 'dhcp::dhcp6' do
+  on_os_under_test.each do |os, facts|
+    context "on #{os}" do
+      let :facts do
+        facts
+      end
+
+      let :pre_condition do
+        "class { '::dhcp': interfaces => ['eth0']}"
```

Comment:
Ideally we wouldn't need this.

Reply:
I'll fix this, but it will require the packagename to be left in dhcp::dhcp6.